backend/person/models.py

```diff
@@ -1,28 +1,108 @@
 from django.db import models
-from core.models import Field
+from core.models import Field, LettercraftDate
+
+
+class Office(models.Model):
+    """
+    A job or position that a person can hold.
+    """
+
+    name = models.CharField(
+        max_length=256,
+        help_text="The name of the office (e.g. 'bishop of Toulouse', 'king of France')",
+    )
+    description = models.TextField(
+        blank=True,
+        help_text="A description of the office (e.g. 'The bishop of Toulouse is the ecclesiastical ruler of the diocese of Toulouse')",
+    )
+
+    def __str__(self):
+        return self.name
```

Comment:
My impression is that  "office" is a good term to describe the position of bishops or kings but an odd term for "regular" occupations (e.g. shepherd).

This isn't an issue if it reflects what the researchers have in mind, but maybe we should check that in the next meeting.

Reply:
The 'office of shepherd' sounds a bit off, indeed :P Let's ask the team on Wednesday whether they have a better solution.